### PR TITLE
Docker environment variable changes.

### PR DESCRIPTION
**Pull request by rwblair**
Have the code read mongo and api settings from environment variables passed into the containers from a file called .env

### CODE REVIEW

File: dockereve-master/docker-compose.yml [hunk 1]
```diff
@@ -13,11 +13,11 @@ nginx:
 
 swagger_ui:
     image: swaggerapi/swagger-ui:v3.0.12
-    environment:
-        - API_URL=http://${HOSTNAME}/docs/api
```
Comment (chrisgorgo) [inline dockereve-master/docker-compose.yml:17]: how come you can skip this?

File: dockereve-master/docker-compose.yml [hunk 2]
```diff
@@ -13,11 +13,11 @@ nginx:
...
+    env_file: .env
```
Comment (chrisgorgo) [inline dockereve-master/docker-compose.yml:16]: I don't think you need to do this explicitly. See: https://docs.docker.com/compose/environment-variables/#the-env-file

**Review by chrisgorgo — CHANGES_REQUESTED** [review]
I'm afraid this would break Swagger UI.

File: dockereve-master/nginx/sites-enabled/eve_project [hunk 3]
```diff
@@ -1,7 +1,7 @@
 server {
 
     listen 80;
-    server_name example.org;
+    server_name _;
```
Comment (oesteban) [inline dockereve-master/nginx/sites-enabled/eve_project:4]: mriqc.nimh.nih.gov ?

File: dockereve-master/.env [hunk 4]
```diff
@@ -0,0 +1,5 @@
+MONGODB_HOST=mongodb
+MONGODB_PORT=27017
+
+API_TOKEN=afaketoken
+API_URL=http://example.com/url/to/docs
```
Comment (chrisgorgo) [inline dockereve-master/.env]: Actually if you change this to `http://localhost/docs/api things should work out of the box in dev.

**Pull request merged**